server/views/pages/recall/recall-recorded.njk

```diff
@@ -0,0 +1,29 @@
+{% extends "../../partials/prisonerBannerLayout.njk" %}
+
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+
+
+{% block content %}
+    <div class='govuk-grid-row'>
+        <div class='govuk-grid-column-two-thirds'>
+
+          {{ govukPanel({
+            titleText: "Recall recorded"
+          }) }}
+
+          <div class="govuk-!-margin-top-6">
+            <h2 class="govuk-heading-l">What you can do next</h2>
+
+            <p><a href="{{ urls.adjustments }}">Check the adjustments information</a></p>
```

Comment:
does a recall user always have the adjustments role? I assume you'd only want to show the adjustment link to users who can use it?

Reply:
Have picked up with BAs - the application object in config is an object to allow for the required roles to be listed if that's the case